confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/storagegroup/DatabaseSchemaPlan.java

```diff
@@ -28,17 +29,17 @@
 import java.nio.ByteBuffer;
 import java.util.Objects;
 
-public class SetStorageGroupPlan extends ConfigPhysicalPlan {
+public class DatabaseSchemaPlan extends ConfigPhysicalPlan {
```

Comment:
Do we need some verb such as `set` or `update`?

Reply:
I suppose no. Because the DatabaseSchemaPlan will be used in CreateDatabase or AlterDatabase. Both verb `set` or `update` can't not match these two scenarios in the same time.

Reply:
OK